source/guide/3.1/plugin-docker.md

```diff
@@ -189,3 +191,7 @@ Here are the operations that this plugin currently supports:
 ### Delete task
 
 * Deletes the container and its runtime_properties.
+
+# Complete Example
+
+For a complete working example, please see the [ReadMe](https://github.com/cloudify-cosmo/cloudify-nodecellar-docker-example/tree/3.1) for the Docker Nodecellar Example.
```

Comment:
Use the global placeholder {{site.latest_cloudify_version}} placeholder here. You can remove the cloudify_version property at the top.